src/extra_checks/checks/model_field_checks.py

```diff
@@ -305,8 +305,10 @@ def apply(
         choices = field.flatchoices  # type: ignore
         if choices:
             field_choices = [c[0] for c in choices]
-            if field.blank and "" not in field_choices:
+            if field.blank and "" not in field_choices and field.empty_strings_allowed:
```

Comment:
This looks like real problem, can you describe an example? Or even add test here https://github.com/kalekseev/django-extra-checks/blob/master/tests/test_model_field_checks.py

Reply:
I'll add a test, but take for instance this field:
`integer_blank = models.IntegerField(choices=[(1, "One"), (2, "Two")], blank=True, null=True)`
As an optional field, we want to keep `blank=True` for Django admin for instance, but doing so would add an empty string to the suggested constraint:
`check=models.Q(integer_blank__in=[1, 2, ""]`
While the good constraint would be `check=models.Q(integer_blank__in=[1, 2, None]` or just `check=models.Q(integer_blank__in=[1, 2]` per your previous comment.

Hence, we just check `empty_strings_allowed`, but could also check manually that the added value is accepted in this field (maybe an idea for another constraint check?).

Reply:
@kalekseev added some tests, there does not seem to be any regression in the test suite. Thanks for your great reactivity!